examples/ai-billing/auth.ts

```diff
@@ -0,0 +1,37 @@
+import NextAuth from "next-auth";
+
+import GitHub from "next-auth/providers/github";
+
+import type { NextAuthConfig } from "next-auth";
+
+export const config = {
+  pages: {
+    signIn: "/sign-in",
+  },
+  theme: {
+    logo: "https://next-auth.js.org/img/logo/logo-sm.png",
```

Comment:
no idea how this shows up, can we just replace it like this?
```suggestion
    logo: "https://unkey.dev/images/landing/unkey.png",
```